src/models/stack.js

```diff
@@ -148,10 +148,6 @@ for (const method of EVENT_HANDLER_METHODS) {
   Stack.prototype[method] = function (state, editor, ...args) {
     debug(method)
 
-    if (method == 'onChange') {
-      state = this.onBeforeChange(state, editor)
```

Comment:
This `onBeforeChange` was never called since `"onChange"` is in `STATE_ACCUMULATOR_METHODS` and not `EVENT_HANDLER_METHODS`.